src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3580,7 +3580,7 @@ void MusicXMLParserDirection::handleFraction()
 
     for (size_t n = 0; n < fracs.size(); n++) {
         if (rawWordsText.contains(fracs.at(n))) {
-            int p = n <= 2 ? 0x00BC + n : 0x2150 + n - 3;
+            size_t p = n <= 2 ? 0x00BC + n : 0x2150 + n - 3;
```

Comment:
I think you can just use `char16_t` here directly. Arithmetics also work for that type.
And then you can of course remove the cast in the next line, making it just `String(p)`.

Reply:
I thought so too, so I tried. It didn't work.

Reply:
It is the `n`. Casting that into a `char16_t`might be an option. But it won't work without a cast, and then twice...

Reply:
I see. Let's go with this change then, it's for sure an improvement.